pgserver/connection_handler.go

```diff
@@ -1045,9 +1045,12 @@ func (h *ConnectionHandler) query(query ConvertedQuery) error {
 	}
 
 	callback := h.spoolRowsCallback(query.StatementTag, &rowsAffected, false)
-	err := h.duckHandler.ComQuery(context.Background(), h.mysqlConn, query.String, query.AST, callback)
-	if err != nil {
-		return err
+	if query.SubscriptionConfig != nil {
+		return executeCreateSubscriptionSQL(h, query.SubscriptionConfig)
+	} else {
```

Comment:
Use 'else if' may be better for readability and clarity.

Reply:
fixed